js/clinicalTimeline.js

```diff
@@ -156,7 +155,7 @@ var clinicalTimeline = (function(){
     d3.select(divId + " svg")
       .insert("rect", ".timeline-label")
       .attr("width", 130)
-      .attr("height", gBoundingBox.height - 15)
+      .attr("height", 245)
```

Comment:
This should probably be a constant, since I see the same size being used here: https://github.com/rohangoel96/clinical-timeline/blob/591ba394b8eacf5cdc4d63aafa75efc179032a41/js/clinicalTimeline.js#L158